repos/fdbt-netex-output/src/netex-convertor/period-tickets/periodTicketNetexHelpers.ts

```diff
@@ -1052,7 +1052,11 @@ export const getPeriodConditionsElement = (
                             ? 'endOfFareDay'
                             : 'standardDuration',
                 },
-                ActivationMeans: { $t: 'noneRequired' },
+                ActivationMeans: product.productValidity === 'fareDayEnd' ? undefined : { $t: 'noneRequired' },
+                EndTime:
+                    product.productValidity === 'fareDayEnd' && 'fareDayEnd' in userPeriodTicket
```

Comment:
If the productValidity is endOfCalenderDay, I think we should be setting EndTime to 2359